lib/ts/recipe/passwordless/components/features/signInAndUp/index.tsx

```diff
@@ -161,7 +164,29 @@ class SignInUp extends PureComponent<PropType, SignInUpState> {
 
     componentDidMount = async (): Promise<void> => {
         const loginAttemptInfo = await this.getModifiedRecipeImplementation().getLoginAttemptInfo();
-        this.setState((s) => ({ ...s, loaded: true, loginAttemptInfo }));
+
+        // We could be using storage events for this, but we need to keep customization in mind.
+        // Someone could be using something else other than localstorage.
+        const checkSessionIntervalHandle = setInterval(async () => {
+            // TODO: check if we can call into the session recipe from here.
+            const hasSession = await Session.doesSessionExist();
+            if (hasSession) {
+                return this.props.recipe.redirect(
+                    {
+                        action: "SUCCESS_IN_ANOTHER_TAB",
+                    },
+                    this.props.history
+                );
```

Comment:
I don't think it makes sense to make this its own page. I say this cause it implies then that someone can navigate to it and see the UI there, which doesn't make sense from a user flow point of view. I'd rather have it as another component within this feature component (just how enter phone, and enter user input code are).

That being said, i'd still keep the user config and overridability of this component as is in this PR.